protected/humhub/modules/admin/grid/SpaceActionColumn.php

```diff
@@ -10,8 +10,6 @@
 
 use Yii;
 use humhub\libs\ActionColumn;
-use humhub\modules\space\models\Space;
```

Comment:
`$model` = Space class call

Reply:
To be investigated. Can not find it. Are we using different versions of Humhub?

Reply:
Is this the best way to solve this problem?  How do I know the relation between Space and $model?